driver/src/http.rs

```diff
@@ -75,22 +94,165 @@ impl HttpClient {
     }
 
     /// Standard HTTP GET request that parses the result as JSON.
-    pub fn get_json<U, T>(&self, url: U) -> Result<T>
+    fn get_json<U, T>(&self, url: U) -> Result<(T, Duration, usize)>
```

Comment:
I'm not sure I'm a fan of this abstraction as it is leaking pretty obviously that the size and duration are for measurement purposes. What the advantage of wrapping this into two components over adding metric hooks directly into the inner client?

Reply:
Good point! that was left-over from when everything had labeled and unlabeled equivalents. I'll fix it.